azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -151,13 +152,15 @@ public KubernetesContainerizedImpl(final Props azkProps,
       final ExecutorLoader executorLoader,
       final VersionSetLoader versionSetLoader,
       final ImageRampupManager imageRampupManager,
-      final KubernetesWatch kubernetesWatch)
+      final KubernetesWatch kubernetesWatch, EventListener eventListener)
       throws ExecutorManagerException {
     this.azkProps = azkProps;
     this.executorLoader = executorLoader;
     this.versionSetLoader = versionSetLoader;
     this.imageRampupManager = imageRampupManager;
     this.kubernetesWatch = kubernetesWatch;
+    this.eventListener = eventListener;
+    this.addListener(this.eventListener);
```

Comment:
Don't add the listener here again. It will fire two events. It should be added only once which is happening from ContainerizedDispatchManager.

Reply:
removed listener